scripts/report_scripts/_settings.Rmd

```diff
@@ -0,0 +1,34 @@
+
+Reading Config File: 
+
+```{r}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE, message = FALSE)
+suppressMessages(suppressWarnings(library(knitr)))
+suppressMessages(suppressWarnings(library(ggplot2)))
+#suppressMessages(suppressWarnings(library(ggrepel)))
+suppressMessages(suppressWarnings(library(data.table)))
+#suppressMessages(suppressWarnings(library(rtracklayer)))
+suppressMessages(suppressWarnings(library(plotly)))
+#suppressMessages(suppressWarnings(library(GenomicRanges)))
+#suppressMessages(suppressWarnings(library(parallel)))
+#suppressMessages(suppressWarnings(library(pbapply)))
+suppressMessages(suppressWarnings(library(DT)))
+suppressMessages(suppressWarnings(library(dplyr)))
+suppressMessages(suppressWarnings(library(tidyr)))
+suppressMessages(suppressWarnings(library(qpcR)))
+suppressMessages(suppressWarnings(library(stringr)))
+suppressMessages(suppressWarnings(library(magrittr)))
+suppressMessages(suppressWarnings(library(openssl)))
+
+if (exists("params")){
+  sampleName <- params$sample_name
+}
+config <- yaml::read_yaml('./config.yml')
```

Comment:
This is likely going to lead to problems down the line; is it guaranteed that the config.yml (should be "yaml") ends up in the same directory?  Is the relative file name resolved relative to this file or the current working directory?  They will not be the same when the pipeline is installed.

Better accept the config file location as an argument.

Reply:
the _settings.Rmd and the config.yml are getting copied in the same folder... This is basically the header for each report.Rmd to read in the config files parameters